modules/journal.js

```diff
@@ -0,0 +1,96 @@
+const axios = require('axios')
+
+// mock modules
+const mockjournals = []
+const mocklwd = []
+const mockdes = []
+
+const timeout = (ms) => new Promise((resolve) => setTimeout(resolve, ms))
+const getJournals = async () => {
```

Comment:
you could try to use https://github.com/EQWorks/avail-bot/blob/8541e4ec3f16c1982f8e6c9d51e58c8f1a1b338a/src/availability.js#L47, providing `projectName: 'Dev Journal` and whatever fields you want in `rawParams.opt_fields`